<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>811</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableView" name="journalTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>781</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Сортировка</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sortJournalComboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteJournalButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить запись</string>
      </property>
     </widget>
     <widget class="QPushButton" name="insertJournalButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Вставить запись</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editJournalButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>360</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Редактировать запись</string>
      </property>
     </widget>
     <widget class="QPushButton" name="returnedJournalButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>360</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Прибыл</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="deletePersonnelButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить запись</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editPersonnelButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>360</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Редактировать запись</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sortPersonnelComboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Сортировка</string>
      </property>
     </widget>
     <widget class="QTableView" name="personnelTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>781</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="insertPersonnelButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Вставить запись</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="deleteAutoButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить запись</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editAutoButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>360</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Редактировать запись</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sortAutoComboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Сортировка</string>
      </property>
     </widget>
     <widget class="QTableView" name="autoTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>781</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="insertAutoButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Вставить запись</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="deleteRouteButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить запись</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editRouteButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>360</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Редактировать запись</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sortRoutesComboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Сортировка</string>
      </property>
     </widget>
     <widget class="QTableView" name="routesTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>781</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="insertRouteButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Вставить запись</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QTableView" name="additionalTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>781</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="additionalComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>361</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="additionalRouteComboBox">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>396</x>
        <y>10</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Маршрут:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
